lib/json/static.nit

```diff
@@ -111,10 +111,8 @@ redef class Text
 				else if char == 0x08.code_point then
 					buffer.append "\\b"
 				else
-					buffer.append "\\u000{char.code_point.to_hex}"
+					buffer.append "\\u00{char.code_point.to_hex}"
```

Comment:
shouldn't you take in account the size of the hex value so that there is exaclty 4 hex digits?